.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,15 +3,15 @@
 | Branch?       | master for features / 2.7 up to 4.0 for bug fixes <!-- see below -->
 | Bug fix?      | yes/no
 | New feature?  | yes/no <!-- don't forget to update src/**/CHANGELOG.md files -->
-| BC breaks?    | yes/no
+| BC breaks?    | no     <!-- see https://symfony.com/bc -->
 | Deprecations? | yes/no <!-- don't forget to update UPGRADE-*.md files -->
-| Tests pass?   | yes/no
-| Fixed tickets | #... <!-- #-prefixed issue number(s), if any -->
+| Tests pass?   | yes    <!-- please add some, will be required by reviewers -->
+| Fixed tickets | #...   <!-- #-prefixed issue number(s), if any -->
 | License       | MIT
-| Doc PR        | symfony/symfony-docs#... <!--highly recommended for new features-->
+| Doc PR        | symfony/symfony-docs#... <!-- required for new features -->
```

Comment:
This isn't realistic. Do you want to force users to write docs for ever little new feature (like mine here https://github.com/symfony/symfony/pull/26218/files)?

Reply:
this is only a template, "force" is a bit too strong in your sentence :)
but yes, we want to ensure contributors wonder seriously about the doc PR
then, common sense will drive reality, as usual :)

Reply:
@ostrolucky I agree: it's not realistic to ask that code contributors write the full docs of their features (besides, most developers can't write good docs). But we'll no longer accept pull requests with empty or very vague descriptions.

Instead, pull requests must explain (very briefly) what they do or improve or change. Also, when applicable, we'd need some simple code or config example for the feature. With that information, members of the Doc Team can write the definitive docs.

Reply:
and if it is not the documentation PR, most features at least need a doc *issue* to make the doc team aware of the need to document this